### ISSUE TITLE
System.TypeLoadException after using VS test runner

### ISSUE THREAD

**Issue by llehn** [xunit/xunit#293]
I have integration test giving me this error on Win7 / VS2013 / .NET 4.5 (tried 3 different machines) using xunit 2 rc1 as well as xunit 1.9.2 (not at the same time, but both give this error)

     System.TypeLoadException : The method "CreateRequestContainer" in Type  
     "My.Web.Host.Bootstrapper" in the assembly "My.Web.Host, Version=1.0.0.0, Culture=neutral, 
     PublicKeyToken=null" has no implementation.

My test code looks like that:

````c# 
using (var bootstrapper = new Bootstrapper(
               Persistence.PersistenceMode.Memory, 
               new FixturePopulation()))
     {
            ....
````

The Bootstrapper class is custom implemented derived from Nancy.Bootstrappers.Autofac.NancyAutofacBootstrapper. There's also nhibernate initialization there (all wired by autofac). 

The thing is, I only get this error when running the tests from the VS test explorer. When using the command line runner, I only get this error when I had a previous run using VS test explorer.

Basically it's this way:
-clean, build
-run console runner -> not getting that error
-run vs runner -> error
-run console runner -> error

Randomly it seems to work from the VS test runner, but only like 1, 2 times then it stops working and the error comes again.

When I break at this exception in debugger, all required assemblies seem to be loaded into the current AppDomain, but entering 
````c# 
typeof(My.Web.Host.Bootstrapper)
````
in the watch window doesn't give any result, but "helper threw an exception"

**Comment by bradwilson** [comment]
Do you have a repro case?

**Comment by llehn** [comment]
I found the cause while trying to strip down the project to create a minimal repro case. I had 2 projects using Nancy, but vastly different versions, which caused this problem, because one project was compiled against one version but run and tested against the other - what a fuckup, my bad. Issue can be closed.
